src/GraphQLParser/Visitors/DefaultNodeVisitor.cs

```diff
@@ -16,6 +16,7 @@ public class DefaultNodeVisitor<TContext> : INodeVisitor<TContext>
     /// <inheritdoc/>
     public virtual async ValueTask VisitDocument(GraphQLDocument document, TContext context)
     {
+        await Visit(document.Comment, context).ConfigureAwait(false); // Comment always null
```

Comment:
Here I wanted to show explicitly that I did not forget to visit comment node even if it always null.